.ci/setup_env_sles.sh

```diff
@@ -55,6 +55,9 @@ chronic sudo -E zypper -n install libudev-devel
 echo "Install Build Tools"
 chronic sudo -E zypper -n install -t pattern "Basis-Devel" && sudo -E zypper -n install python zlib-devel
 
+echo "Install YAML validator"
+chronic sudo -E zypper -n install yamllint
```

Comment:
@jodh-intel that does not exist on sles or opensuse, this is the CI error
```
'yamllint' not found in package names. Trying capabilities.
```

Reply:
stepping back.... the intention is to test that our label tooling is adequate.  There really shouldn't be a need to test this on a per-distro basis -- just in a single job would be enough.

Is there a way to skip this for openSUSE?  WDYT?

Reply:
Fixed.

Reply:
now python3-pip not found on SLES:
`04:18:37 Package 'python3-pip' not found.
`

Reply:
@marcov, @GabyCT - I don't have access to a SLES system. Can you tell me the name of the pip package for sles? `python2-pip` or `python-pip` maybe?

Reply:
@jodh-intel 
```
$ zypper -n install python2-pip
```

Reply:
@egernst - You are correct. At this stage, we're just checking the single db file so this check could be summarised as:

- > Only run this when one of the db files changes.

But in the future, the plan is to add extra checks on the labels on the PR and maybe even its associated issue too. In fact, we may end up adding labels. At that point it would become a:

- > Run **exactly once** per PR.

That implies of course that it should not run on every CI system.

I think we're still waiting for the ability to run arbitrary periodic jobs under Jenkins but this should be considered as currently it will be running "too many times".

Until we can move it though, does anyone know how to install pip on SLES 12? If I don't find out by Monday, I'll have to disable for SLES which would be a shame as all the distros should have the ability to run all tests *in principle*.

Reply:
> @jodh-intel
> 
> ```
> $ zypper -n install python2-pip
> ```

Reply:
Grr - browser cache fail. Branch updated to use `python2-pip` for sles now.